modules/ding_facetbrowser/ding_facetbrowser.module

```diff
@@ -138,7 +138,7 @@ function ding_facetbrowser_form($form, &$form_state, $facets = FALSE, $search =
       if (count($facet->terms) > 0) {
         $form[$facet->name] = array(
           '#type' => 'fieldset',
-          '#title' => check_plain($configured_facet['title']),
+          '#title' => t(check_plain($configured_facet['title'])),
```

Comment:
Are you aware that source-language of these facets are Danish?

Reply:
Yes. Is that a problem?

Reply:
When you add `t()` around like this, I suppose the source language will be English? So it will be a Danish string in the English field.